lib/modules/announcements.js

```diff
@@ -30,53 +31,70 @@ const recheckPostAfter = HOUR;
 
 const now = Date.now();
 
-module.go = () => {
+let post;
```

Comment:
I don't really like module globals like this, I think we could call `postForNotification()` in `go()` (it doesn't need to be super fast in `beforeLoad`) and pipe everything down through arguments